front/app/services/permissions/rules/routePermissions.ts

```diff
@@ -14,7 +14,6 @@ import { IAppConfigurationData } from 'services/appConfiguration';
 const MODERATOR_ROUTES = [
   '/admin/dashboard',
   '/admin/projects',
-  '/admin/messaging',
```

Comment:
Main change.

Reply:
Will it be possible for moderators to see messages?

Reply:
No, see ticket. Access to /admin/messaging is shut off for now. This means custom, automatic email campaigns as well as text campaigns.